src/renderer/schemas/Address.schema.ts

```diff
@@ -0,0 +1,7 @@
+export interface Address {
+  id: string;
+  name: string;
+  walletAddress: string;
+  createdAt: Date;
```

Comment:
we dont need createdAt and updatedAt in the UI. these values should be determined by the API, not dictated by the UI